### PR TITLE
Merge to master PR

### PR DESCRIPTION
**Pull request by wathigo**

### CODE REVIEW

File: css/styles.css [hunk 1]
```diff
@@ -0,0 +1,417 @@
...
+.nav-2::before{
+  position: absolute;
+  content: '';
+  top: 100px;
+  right: 0;
+  width: 66px;
+  display: block;
+  background-color: #f72210;
+  border-left: 13px solid #ffff;
+  border-top: 12px solid #f72210;
```
Comment (azdrenymeri) [inline css/styles.css:78]: `border-top: 12px solid #f72210;` is unnecessary the top border color is set by `background-color: #f72210;` on line 77.

File: css/styles.css [hunk 2]
```diff
@@ -0,0 +1,417 @@
...
+  font-size: 0.9em;
+  font-weight: bold;
+}
+.featured-stories-wrapper{
+  padding-top: 15px;
+}
+.featured-stories figure, .top-story figure{
+  position: relative;
+}
+.featured-stories p:first-of-type2{
```
Comment (azdrenymeri) [inline css/styles.css]: `:first-of-type2` is an unknown selector

**Review by azdrenymeri — COMMENTED** [review]
## Hello @wathigo :wave: 
* You have some whitespace(Here is how it looks on my screen)
screenshot
#### Implement the required changes and add it for a re-review

**Comment by wathigo** [comment]
Hello @azdrenymeri . Thank you for your feedback. I have been able to fix the last two issues you mentioned. As for the first one on white space, everything renders well on my browser.
Screenshot from 2019-05-27 04-41-13
Screenshot from 2019-05-27 04-41-59

File: css/styles.css (hunk 1 above)
Comment (wathigo) [reply css/styles.css]: The border top that has the same color with the background-color is not by mistake. This is in order to achieve the desired effect on the on the nav-2:before.

**Comment by azdrenymeri** [comment]
## It's a :+1:  from me
#### you can merge now

**Comment by wathigo** [comment]
Thank you @azdrenymeri

**Pull request merged**